### ISSUE TITLE
Same outputs after model.eval()

### ISSUE THREAD

**Issue by farsal** [shijianjian/EfficientNet-PyTorch-3D#6]
Hi, I have some issues when using model on validation data, model.eval() gives the same output for any validation data, Can you please help me with this?

**Comment by WinstonHuTiger** [comment]
@farsal did you solve this, I face the same problem.

**Comment by nooppi18** [comment]
I am facing the same problem.

**Comment by pranshu97** [comment]
Did anyone find a solution for this issue?
@shijianjian @farsal @WinstonHuTiger @nooppi18

**Comment by shijianjian** [comment]
Sorry that I did not aware of those issues in this repo. I currently do not actively maintain this repo. So, if you have resolved this, drop me a PR and at me. I will merge.

I do not check much on the code. Presumable, if ```model.train()``` worked but not ```model.eval()```, I suspect something went wrong with train/test conditional layers as BatchNorm, dropout, etc. Good luck!

**Comment by AmokraneMancer** [comment]
same issue, did any one find a solution ?

**Comment by yamazakiji** [comment]
please disable `model.eval()` and  use `torch.no_grad()`
